docker-compose.yml

```
@@ -3,22 +1,43 @@
+    ports:
+      - 80:80
+      - 443:443
+    depends_on:
+      - mongodb
+  mongodb:
+    image: docker.io/library/mongo:8.0.0-noble
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: openwfagent
+      MONGO_INITDB_ROOT_PASSWORD: spaceninjaserver
+    volumes:
+      - ./data/database:/data/db
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)